### ISSUE TITLE
Question: validation set size, how many epochs to train?

### ISSUE THREAD

**Issue by domcross** [rishikksh20/VocGAN#10]
What is the recommend size of the validation set for a data set with 22000 wav files?

How many epochs/iterations to train for a feasible result? How many epochs did you train the model used on the demo site?

**Comment by rishikksh20** [comment]
@domcross Ideally, validation size is 5 % of your total data point but vocoder  performance usually subjective to listening only so 5 to 10 number of files would be fine. I generally use 5 files as validation.
Make sure batch size should always 16, and training around 800k step (around 600 epochs ) starting giving good results.

**Comment by domcross** [comment]
Thanks.
